### PR TITLE
Compile and overwrite test

### PR DESCRIPTION
**Pull request by judovana**

### CODE REVIEW

**Comment by AurumTheEnd** [comment]
I would suggest moving the **"utility"** classes to a different package in order to differentiate them from the actual org.jrd.backend.data **test** classes (`CliTest`, `CompileUploadCliTest`, but also `ArchiveManagerOptionsTest`).

File: .github/workflows/maven.yaml [hunk 1]
```diff
@@ -76,4 +76,4 @@ jobs:
       - name: Test Cli
         run: |
           cd jrd
-          mvn --batch-mode test -Dtest=CliTest -DfailIfNoTests=false
+          mvn --batch-mode test -Dtest=*CliTest -DfailIfNoTests=false
```
Comment (judovana) [inline .github/workflows/maven.yaml:80]: Why there is this condition at all? Test all!
Also the -DfailIfNoTests=false seems wrong. lets fail

**Comment by judovana** [comment]
> I see your 20 minute timeout still didn't help on Windows. Since all tests finish in under a minute on Ubuntu, this is a more serious issue than Windows just being slightly slower (which was the case prior to this PR - Windows build took about 1.5x the total time of the Ubuntu one).

yy. remopved

File: runtime-decompiler/src/main/java/org/jrd/backend/data/Cli.java [hunk 2]
```diff
@@ -262,13 +268,28 @@ private void overwrite() throws Exception {
         }
     }
 
+    @SuppressFBWarnings(value = "OS_OPEN_STREAM", justification = "The stream is clsoed as conditionally as is created")
```
Comment (AurumTheEnd) [inline runtime-decompiler/src/main/java/org/jrd/backend/data/Cli.java:271]: Probably could be *cheesed* by having the condition as a `final boolean` variable. Probably.

**Comment by AurumTheEnd** [comment]
> Please fix in a separate PR.

Nah. :)

**Pull request merged**